static/js/app.ts

```diff
@@ -188,6 +188,21 @@ function clearErrors(editor: AceAjax.Editor) {
   }
 }
 
+export function newly_learned_subjects(level: any, code: string) {
```

Comment:
Can we change this too:

```suggestion
export function newly_learned_commands(level: any, code: string) {
```

(and of course make the same change in other relevant places)